compiler/ras/Debug.cpp

```diff
@@ -4204,6 +4204,8 @@ TR_Debug::getRuntimeHelperName(int32_t index)
          case TR_ARM64arrayCopy:                                   return "__arrayCopy";
          case TR_ARM64forwardArrayCopy:                            return "__forwardArrayCopy";
          case TR_ARM64backwardArrayCopy:                           return "__backwardArrayCopy";
+         case TR_ARM64interfaceCompleteSlot2:                      return "_interfaceCompleteSlot2";
```

Comment:
Should this be "Compete" rather than "Complete" ?

Reply:
After reviewing the downstream code, it looks like "Complete" is the right word.